lib/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -188,13 +189,16 @@ private <T> T getJsonProperty(
             JsonNode jsonNode,
             String propertyName,
             String credentialIssuer,
-            CollectionType valueType)
+            CollectionType valueType,
+            boolean validateCorrelation)
```

Comment:
Would it be clearer to make this more generic, something like `shouldThrowErrorIfMissing`?

Reply:
I used the validateCorrelation that we decided together on in today's call. I think this is more appropriate as it not only throws an exception, but also returns an empty list. There is business according this param in method not only error message. Therefore, I think validateCorrelation is more suitable.